core/templates/pages/skill-editor-page/skill-editor-page.component.ts

```diff
@@ -96,6 +104,78 @@ angular.module('oppia').component('skillEditorPage', {
         return ctrl.skill ? ctrl.skill.getValidationIssues().length : 0;
       };
 
+      ctrl.onClosingSkillEditorBrowserTab = function() {
+        const skill = SkillEditorStateService.getSkill();
+
+        const skillEditorBrowserTabsInfo: EntityEditorBrowserTabsInfo = (
+          LocalStorageService.getEntityEditorBrowserTabsInfo(
+            EntityEditorBrowserTabsInfoDomainConstants
+              .OPENED_SKILL_EDITOR_BROWSER_TABS, skill.getId()));
+
+        if (skillEditorBrowserTabsInfo.doesSomeTabHaveUnsavedChanges() &&
+            UndoRedoService.getChangeCount() > 0) {
+          skillEditorBrowserTabsInfo.setSomeTabHasUnsavedChanges(false);
+        }
+        skillEditorBrowserTabsInfo.decrementNumberOfOpenedTabs();
+
+        LocalStorageService.updateEntityEditorBrowserTabsInfo(
+          skillEditorBrowserTabsInfo,
+          EntityEditorBrowserTabsInfoDomainConstants
+            .OPENED_SKILL_EDITOR_BROWSER_TABS);
+      };
+
+      let createSkillEditorBrowserTabsInfo = function() {
```

Comment:
Here and for the 2 methods below, please use const.

Reply:
Done.